integrations/client/test_delphi_epidata.py

```diff
@@ -306,6 +306,24 @@ def test_sandbox(self, get, post):
       Epidata.debug = False
       Epidata.sandbox = False
 
+  @patch('requests.get')
+  def test_version_check(self, get):
+    """Test that in debug + sandbox mode request params are correctly logged, but no queries are sent."""
+    class MockJson:
+      def __init__(self, content, status_code):
+          self.content = content
+          self.status_code = status_code
+      def raise_for_status(self): pass
+      def json(self): return json.loads(self.content)
+    get.reset_mock()
+    get.return_value = MockJson(b'{"info": {"version": "0.0.1"}}', 200)
+    Epidata._version_check()
+    captured = self.capsys.readouterr()
+    output = captured.err.splitlines()
+    self.assertEqual(len(output), 1)
+    self.assertIn("Client version not up to date", output[0])
+    self.assertIn("\'latest_version\': \'0.0.1\'", output[0])
```

Comment:
would it make sense to add another test that tries to re-import the module to ensure the message is not repeated?